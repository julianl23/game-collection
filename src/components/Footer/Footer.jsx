import React from "react";
import styled from "styled-components";

const Footer = () => {
  const Wrapper = styled.footer`
    background: #d8d8d8;
    border: 1px solid #999;
    border-left: 0;
    border-right: 0;
    display: flex;
    align-items: center;
    padding: 10px 25px;
  `;
  const currentYear = new Date().getFullYear();

  return <Wrapper>&copy; {currentYear} Julian Lord</Wrapper>;
};

export default Footer;
