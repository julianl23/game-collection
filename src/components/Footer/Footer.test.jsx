import React from "react";
import TestRenderer from "react-test-renderer";
import Footer from "./Footer";

describe("Footer", () => {
  it("renders correctly", () => {
    const instance = TestRenderer.create(<Footer />);
    expect(instance.toJSON()).toMatchSnapshot();
  });
});
