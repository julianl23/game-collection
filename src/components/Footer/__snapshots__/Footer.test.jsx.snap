// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
.c0 {
  background: #d8d8d8;
  border: 1px solid #999;
  border-left: 0;
  border-right: 0;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding: 10px 25px;
}

<footer
  className="c0"
>
  Â© 
  2019
   Julian Lord
</footer>
`;
